Dockerfile

```diff
@@ -1,17 +1,19 @@
-FROM golang:alpine as builder
+FROM golang:1.15-alpine AS builder
 
-WORKDIR /go/src/github.com/blendle/pg2kafka
-ADD . ./
+# needed for gcc
+RUN apk add --no-cache build-base && \
+    apk add --no-cache git && \
+    apk add --no-cache --upgrade bash
 
-RUN apk --update --no-cache add git alpine-sdk bash
-RUN wget -qO- https://github.com/edenhill/librdkafka/archive/v0.11.4-RC1.tar.gz | tar xz
-RUN cd librdkafka-* && ./configure && make && make install
-RUN go get github.com/golang/dep/cmd/dep && dep ensure -vendor-only
-RUN go build -ldflags "-X main.version=$(git rev-parse --short @) -s -extldflags -static" -a -installsuffix cgo .
+WORKDIR /build
+COPY . .
+
+RUN go build -tags musl -ldflags "-X main.version=$(git rev-parse --short @) -s -extldflags -static" -a -installsuffix cgo .
```

Comment:
`main.version=$(git rev-parse --short @)` is used so that git version is logged and correlated by the service.

when we introduce CICD that will build, tag and push docker image, we won't need this as code tag will be aligned with docker image tag.

Reply:
OK let's go with this for now. I do not expect too many changes on this container but we can automate its generation in the future. You can also simply push the current version you have in the codebgp public docker placehodler for pg2kafka.

Reply:
Moved to #2